public/app/plugins/datasource/loki/annotations_query_ctrl.tsx

```diff
@@ -0,0 +1,14 @@
+export class LokiAnnotationsQueryCtrl {
```

Comment:
The underscore file naming is something we are migrating away from and is not for angular components. So name file after primary export :)